docs/versioned_docs/version-3.5/manual/sortview.md

```diff
@@ -33,3 +33,24 @@ You can control what data you view in the grid. On each column, you will see a <
 You can find, under <span class="menuItems">All</span> → <span class="menuItems">View</span>, the option to show and hide [“null” values](exploring#data-types). A small grey “null” will appear in each applicable cell. Remember that a null cell is not the same thing as an empty cell. 
 
 ![A screenshot of what a null value looks like.](/img/null.png)
+
+## Goto {#goto}
```

Comment:
```suggestion
## Page navigation {#pagenavigation}
```

Reply:
Yes. To be consistent with the rest, I would even propose `#page-navigation` as a hash key.